docs/tutorials/migrations/homebrew.md

```diff
@@ -0,0 +1,156 @@
+
+* [`search`][search] is used to find available packages
+* [`install`][install] is used to install packages
+* [`uninstall`][uninstall] is used to remove packages
+
+### Show top-level formulae in Homebrew
+First, identify the list of Homebrew packages you have installed.
+
+We recommend using `brew leaves` for this, so you can easily differentiate between the formulae you installed explicitly versus those that were installed as dependencies. The `leaves` subcommand will show the formulae that were directly installed.
+
+For example:
+
+```
+% brew leaves
+boost
+gh
+imagemagick
+mysql
+openssh
+```
+
+### Search for a package in Flox
+Then, search Flox for one of the formulae you have installed with Homebrew. In this case, for example, you could choose `boost`:
+
+```
+% flox search boost
+boost     Collection of C++ libraries
+booster   Fast and secure initramfs generator
+xgboost   Scalable, Portable and Distributed Gradient Boosting (GBDT, GBRT or GBM) Library
+boost155  Collection of C++ libraries
+boost159  Collection of C++ libraries
+boost15x  Collection of C++ libraries
+boost160  Collection of C++ libraries
+boost165  Collection of C++ libraries
+boost166  Collection of C++ libraries
+boost167  Collection of C++ libraries
+
+Showing 10 of 120 results. Use `flox search boost --all` to see the full list.
+
+Use 'flox show <package>' to see available versions
+```
+
+The first one on the list is the correct Flox package to install, and it has the same name as the Homebrew package. This will often be the case, but not always.
+
+### Install your first package
+To install your first package, use `flox install`:
+
+```
+% flox install boost
+```
+
+The first time you install a package, Flox will ask you whether you want to create a [default environment][default-tutorial]. If you choose to do so, Flox will then ask you whether you want to configure your shell to automatically activate the new default environment.
+
+```
+[show the flow]
+```
+
+When Flox is configured with a default environment, it behaves very similarly to Homebrew. The Flox CLI will assume the default environment when you run `flox install` in a directory that doesn't contain an environment of its own.
+
+Creating a Flox default environment is optional.
+
+If you do not choose for this to be automated at the time of your first package installation, you can [follow these instructions][default-tutorial#setup] to add Flox to your dotfiles manually.
+
+### Verify configuration
+Exit your active shell and create a new one, causing the dotfile changes to take effect. The first time this happens, you may experience a short delay while your default environment is materialized. The next time you open a shell it should be quick because the environment has been cached.
+
+Once the shell is available, you should see `flox [default]` prepended to your shell prompt. You can also verify that your default environment is active by running [`flox envs`][envs]:
```

Comment:
I thought we suppress the prompt for the default env?

Reply:
Agh! you are correct, let me revert that back.